### ISSUE TITLE
Changing needed after incorporating the dashboard

### ISSUE THREAD

**Issue by pkiraly** [pkiraly/qa-catalogue-web#140]
After @NelDav has created the dashboard some changes might be necessary, such as 

- [ ] update the README with the new default for the `default-tab` configuration
- [ ] thinking whether the dashboard needs a menu element
- [ ] minor modifications for the dashboard itself

**Comment by nichtich** [comment]
Just two thoughts:

- The default-tab, if not configured, should not be `start` but `about`
- The class `Start` should be renamed to `Dashboard`

**Comment by pkiraly** [comment]
I agree with the second suggestion. I am not sure about the `about` as starting page, I would expect a page which displays some result instead: validation or completeness. I am also thinking if

- the dashboard should be added to the menu
- which position? at the beginning? before `about`?

Independent, but if we change the menu: `validation` should be placed before `completeness`
